shaded/hadoop/pom.xml

```diff
@@ -142,9 +142,6 @@
                 <relocation>
                   <pattern>com.google</pattern>
                   <shadedPattern>${shading.prefix}.com.google</shadedPattern>
-                  <excludes>
```

Comment:
Does this mean that all of these packages will be shaded now? Why were they not shaded before?

Reply:
actually guice has not been included in hdfs package for a long time, so it did not have an effect. 
not sure why it was shaded before, @LuQQiu do you know?

Reply:
discussed offline, not sure why it was excluded before. but i think we should shade it.